src/js/controllers/walletDetails.js

```diff
@@ -185,9 +231,52 @@ angular.module('copayApp.controllers').controller('walletDetailsController', fun
     });
   };
 
+  $scope.backup = function() {
```

Comment:
In order to complete this you must add a new route in **routes.js** passing walletId as parameter